.github/run_kuttl_tests.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -ex
+
+# Install kuttl
+if [[ ! -f /usr/local/bin/kubectl-kuttl ]]; then
+    wget -O /usr/local/bin/kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.9.0/kubectl-kuttl_0.9.0_linux_x86_64
+    chmod +x /usr/local/bin/kubectl-kuttl
+fi
+
+
+OPERATOR=$(echo ${1} | cut -d '-' -f1)
+export ${OPERATOR^^}_IMG=$2/$1-index:$3
+echo "make ${OPERATOR}_kuttl"
+
+cd $HOME
+rm -rf install_yamls
+git clone https://github.com/openstack-k8s-operators/install_yamls.git
+cd $HOME/install_yamls
+make crc_storage
+sleep 20
+make ${OPERATOR}_kuttl
```

Comment:
As it stands, this would run the kuttl tests from the master branch of the operator. If we want to run the job to test PR changes and they include changes to kuttl tests, they will not be picked up. We should either pass the repo and branch to clone the right branch, or set `KEYSTONE_KUTTL_CONF` and `KEYSTONE_KUTTL_DIR`   to make sure the changes in future PRs are tested